templates/Dockerfile_webservice.template

```diff
@@ -9,13 +9,12 @@ RUN apt-get update \
     software-properties-common \
     telnet \
     vim \
+    wget \
     && apt-get clean
 
-# grab oracle java (auto accept licence)
-RUN add-apt-repository -y ppa:webupd8team/java \
-    && apt-get update \
-    && echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
-    && apt-get install -y oracle-java8-installer
+# grab openjdk
```

Comment:
Double-check that this actually grabs 11, I see `FROM ubuntu:16.04` above
(it also shouldn't float to 12, 13, ...)

Reply:
I originally changed this only to bypass the error that I was getting (develop also gets it).  In any case, I've made another commit to install Java 11.